README.adoc

```diff
@@ -7,7 +7,7 @@ https://app.netlify.com/sites/docs-support-how-to/deploys[image:https://api.netl
 
 toc::[]
 
-This repository contains the source code for https://docs.rackspace.com/support/ website.
+This repository contains the source code for our https://docs.rackspace.com/support/ website.
```

Comment:
FYI this was just typo change to retrigger the build. couldn't track down any comment commands to do it.

Reply:
you can use `--allow-empty` to add an empty commit

```sh
git commit --allow-empty -m "Empty commit to trigger a build"
```